tests/genericmetadata_test.py

```diff
@@ -22,6 +23,78 @@ def test_metadata_overlay(md: comicapi.genericmetadata.GenericMetadata, replaced
     assert md == expected
 
 
+def test_metadata_overlay_overwrite():
+    md = comicapi.genericmetadata.GenericMetadata(series="test", issue="1", title="test", genres={"test", "test2"})
+    overwrite = comicapi.genericmetadata.GenericMetadata(
+        series="test2", issue="2", title="test2", genres={"test3", "test4"}
+    )
+    md.overlay(overwrite, OverlayMode.overwrite)
+
+    assert md == overwrite
```

Comment:
This test makes it obvious that this mode is useless. Just use md.copy()

Reply:
As in anywhere that would have called `md.overlay(new_md, OverlayMode.overwrite)` should instead use `md = new_md.copy()`

Reply:
Because of the commandline clear `-m`, copy can't be used but I'll get into that later. It should be clearer now in the code also.

Reply:
All you've done is implement a type conversion as an overlay mode. Instead do the type conversion in the overlay method or do a manual check when the metadata from the `-m` option is applied